collectors/charts.d.plugin/sensors/metadata.yaml

```diff
@@ -2,20 +2,26 @@ meta:
   plugin_name: charts.d.plugin
   module_name: sensors
   monitored_instance:
-    name: charts.d sensors
+    name: Linux Sensors (sysfs)
     link: ''
-    categories: []
-    icon_filename: ''
+    categories:
+      - data-collection.hardware-devices-and-sensors
+    icon_filename: 'microchip.svg'
   related_resources:
     integrations:
       list: []
   info_provided_to_referring_integrations:
     description: ''
-  keywords: []
+  keywords:
+    - sensors
+    - sysfs
+    - hwmon
+    - rpi
+    - raspberry pi
   most_popular: false
 overview:
   data_collection:
-    metrics_description: ''
+    metrics_description: "Examine Linux Sensors metrics with Netdata for insights into hardware health and performance. Enhance your system's reliability with real-time hardware health insights."
```

Comment:
@Ancairon , please, take a look in next warning.

Reply:
After a talk with Austin, we figured it is safe to ignore for now